### ISSUE TITLE
configs/coco/dcfs_FSIS_r101_novel_shot_seedx.yaml.

### ISSUE THREAD

**Issue by jinse-zyy** [gaobb/DCFS#3]
Hello, I couldn't find "configs/coco/dcfs_FSIS_r101_novel_1shot_seedx.yaml." Could you please provide it to me? Thank you.

**Comment by gaobb** [comment]
hi， 

We thank you for your attention to our work. 

This issue has been fixed by updating config files for training FSIS and gFSIS in the "run_coco_fsis.sh". In fact, the same config file is used to train FSOD and FSIS (gFSOD and gFSIS) in our implements. 

If there is anything, please do not hesitate to let me know.

**Comment by jinse-zyy** [comment]
You're very welcome! While running the experiments, I also noticed some configuration issues in the run_coco_fsod.sh script: 1. The --eval-only option on line 48 is redundant. 2. You need to add the following line of configuration separately for fsod and gfsod: MODEL.ROI_BOX_HEAD.BBOX_CLS_LOSS_TYPE ${classloss}.

In addition, regarding the experimental section in the paper, I have a question about the distinction between "Ours" and "Ours*." You mentioned that "Ours*" is the average of 10 runs, so what does "Ours" represent? Is it the value for the first run?

**Comment by gaobb** [comment]
1.  We train and test simultaneously during the training process, so the test file is not redundant.  https://github.com/gaobb/DCFS/blob/07cad8d0e8ed4239cebb0d1428963264de80241b/run_coco_fsod.sh#L48

2. The loss function is valid using the current command line mode. For example, https://github.com/gaobb/DCFS/blob/07cad8d0e8ed4239cebb0d1428963264de80241b/run_coco_fsod.sh#L46

3.  In the Experimental Details on Page 7 (or the caption of Table 3 on page 8)of our paper, there are detailed descriptions of *Ours* and *Ours\**.

*Following the previous work [33], all experimental results are averaged over 10 seeds. For a fair comparison with DeFRCN [28], we also report the average results over 10 times repeated runs on seed0.*

*The superscript \* indicates that the results are averaged over 10 times repeated runs on seed0, the same as below.*

Here, *Ours* means that the results are averaged over 10 seeds, while *Ours\** implies the result is 10 times repeated runs only on seed0.

**Comment by jinse-zyy** [comment]
Thank you for your response. My previous questions contained errors. The incorrect configuration file is run_voc_fsod.sh.

**Comment by gaobb** [comment]
> Thank you for your response. My previous questions contained errors. The incorrect configuration file is run_voc_fsod.sh.

We greatly appreciate you pointing out this issue, and it has been fixed in *run_voc_fsod.sh*.
